CHANGELOG.md

```diff
@@ -1,3 +1,91 @@
+## streamlink 0.2.0 (12-16-2016)
+
+Release 0.2.0 of Streamlink!
+
+We've done numerous changes to plugins as well as fixed quite a few which were originally failing. Amung these changes are updated docs as well as general UI/UX cleaning with console output.
```

Comment:
hehe I also have two section. My name 'Josip Ponjavic' and my github username 'mmetak'.